xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-picker/xwiki-platform-attachment-picker-webjar/src/main/webjar/attachmentGalleryPicker.js

```diff
@@ -28,30 +28,8 @@ define('xwiki-attachment-picker-translation-keys', {
   ]
 });
 
-define('xwiki-attachment-picker',
-  ['jquery', 'xwiki-attachments-icon', 'xwiki-l10n!xwiki-attachment-picker-translation-keys'],
-  function ($, attachmentsIcon, translations) {
-    'use strict';
-
-    const ATTACHMENT_PICKER_INITIALIZED_CLASS = 'initialized';
-
-    /**
-     * Utility function to debounce an event. The callback is delayed until no similar event is received before the end
-     * of the delay.
-     *
-     * @param callback the function to call when the delay is passed
-     * @param delay the delay in milliseconds before callback is called
-     * @returns {(function(...[*]): void)|*} the function to pass to the event handler
-     */
-    function debounce(callback, delay) {
-      let timeout;
-      return function (...args) {
-        clearTimeout(timeout);
-        timeout = setTimeout(() => callback.apply(this, args), delay);
-      };
-    }
-
-    /**
+define('xwiki-solr-search', ['jquery'], function($) {
```

Comment:
Or `xwiki-attachment-picker-solr-search`, and in this case it could stay here, as iiuc you extracted this code to reuse independently

Reply:
It would make sense to move it to a new `xwiki-platform-attachment-webjar`. I'm not sure I should prioritize this as I never done it and it would take me some time to figure out how to do. Is it okay to move it in the next release if I don't find the time to do it now?